src/components/SelectionList/types.ts

```diff
@@ -297,6 +297,9 @@ type BaseSelectionListProps<TItem extends ListItem> = Partial<ChildrenProps> & {
 
     /** Ref for textInput */
     textInputRef?: MutableRefObject<TextInput | null>;
+
+    /** Whether to hide the list on the initial render. This would prevent the list from "blinking" when you have a long list and it auto scrolls to the bottom on mount but will delay the showing of the list */
```

Comment:
This is better thanks! Here is a suggestion that could improve it a little more:

> When true, the list won't be visible until all items are rendered. This prevents the list from "blinking" as it's scrolled to the bottom which is recommended for large lists. When false, the list will render immediately and scroll to the bottom which works great for small lists.

Reply:
Great. Updated with a little modification.

```diff
-When true, the list won't be visible until all items are rendered.
+When true, the list won't be visible until the list layout is measured
```

because the `isInitialSectionListRender` is set to false in `onLayout` of the list

Reply:
Love it, thanks!